src/Riok.Mapperly/Emit/Syntax/SyntaxFactoryHelper.GeneratedCode.cs

```diff
@@ -0,0 +1,26 @@
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Riok.Mapperly.Emit.Syntax;
+
+public partial struct SyntaxFactoryHelper
+{
+    private const string GeneratedCodeAttributeName = "global::System.CodeDom.Compiler.GeneratedCode";
+    private static readonly AssemblyName _generatorAssemblyName = typeof(SyntaxFactoryHelper).Assembly.GetName();
+
+    public SyntaxList<AttributeListSyntax> GeneratedCodeAttributeList()
+    {
+        var arguments = CommaSeparatedList(
+            AttributeArgument(LiteralExpression(SyntaxKind.StringLiteralExpression, Literal(_generatorAssemblyName.Name))),
```

Comment:
Nitpick: There is a helper for string literals: `StringLiteral(_generatorAssemblyName.Name)`